shared/profile/index.desktop.js

```diff
@@ -139,7 +139,7 @@ class _StellarFederatedAddress extends React.PureComponent<
               style={styles.proofName}
               ref={this.props.setAttachmentRef}
             >
-              <Kb.WithTooltip text={this.props.showingMenu ? '' : 'Stellar Federated Address'}>
+              <Kb.WithTooltip text={this.props.showingMenu ? '' : 'Stellar Federation Address'}>
                 <Kb.Text
```

Comment:
The design still says "federated" doesn't it? is this official nomenclature? https://app.zeplin.io/project/56b25ffc64fc91773a73aaf8/screen/5bdb8ffc5909a6641bb0ab24

Reply:
I think both are fine. Official docs: https://www.stellar.org/developers/guides/anchor/ there is a mention of both "federated address" and "federation address" on the same page. I might have jumped the gun with mass renaming.